### ISSUE TITLE
[Design] Profile: improve address validation UX

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#30258]
In the work we did on #25820, we realized there's more opportunity to improve the UX when we prompt user to review data they entered.  On mobile, a user will press "update" and then be auto-scrolled to the alert that appears at the top of the page. This raises concern for users with motion sensitivity and cognitive considerations.  

Additionally, the "please confirm your address" warning alert box contains two buttons that are not descriptive and are styled as links.

We also received some suggestions from the content team that need further work that isn't included in #25820:
- explore removing the alert, and instead guiding the user directly within the content itself
- or, consider further chunking out the alert content, without making a giant alert that doesn't work great on mobile devices.

Screenshots and suggestions are captured on [this mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1631668096417/8acd03935adaad54b1fae3483d057fcd3b044985?wid=1-1631899151888).

**Comment by andaleliz** [comment]
@ajakabcin FYI this is one of two tickets I created to split out the work we discussed this afternoon.

**Comment by ajakabcin** [comment]
Completing this issue will address #31443

**Comment by andaleliz** [comment]
@Samara-Strauss, I found the [findings summary](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/address-validation/user-testing/Research%20Summary.md) from the address validation research and it looks like this feature was in a modal in profile 1.0 from the screenshots. Was there any specific reason we moved this review/confirmation out of a modal and directly onto the page? We have some similar alerts (essentially asking "are you sure?") in other parts of profile that appear in a modal. Curious to learn what drove that shift.

cc @tressaellen in case you have any thoughts around this from the profile 2.0 design phase.

**Comment by Samara-Strauss** [comment]
TBH, I don't remember the specifics of discussions to move interactions out of modals, and I also don't remember how much thought went into the original decision to use modals in profile 1.0 (I think Mel did that design, so I wasn't the original designer). Tressa's memory might be better than mine. I know the decision to edit on the page instead of in modals for profile 2.0 was very intentional, I just don't remember the specifics of the conversations. Sorry to not be more of a help here :(

**Comment by tressaellen** [comment]
I also don't remember the specifics very well - but I do remember we purposefully introduced in-line editing so users weren't editing _in_ modals. I think the decision to move the review/confirmation out of the modal was part of that decision to try to keep a fluid experience of editing. Honestly I don't think there was any specific reason outside of that. Sorry that I don't have more specifics either - it was all so long ago!

**Comment by andaleliz** [comment]
No problem! I have put some ideas together and shared with Josh and Angela to get a11y feedback before sharing back with the team.  [Here's the comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31443#issuecomment-948659625) in case anyone wants a preview :)

**Comment by andaleliz** [comment]
@MarciMcGuireGCIO I've finalized copy with Danielle, and [written up the spec doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/address-validation/designs/2021-alert-updates/README.md) with mock-ups. This is ready for FE implementation!

I think we need to leave the related a11y ticket open (#31443) until it's actually on production, so I'm going to un-assign myself from that one, and add you for tracking purposes. Please let me know if you have any questions.
